src/main/java/com/algosenpai/app/command/Command.java

```diff
@@ -0,0 +1,19 @@
+package com.algosenpai.app.command;
+
+import com.algosenpai.app.exceptions.DukeExceptions;
+
+public abstract class Command {
+
+    /**
+     * Create new command.
+     */
+    Command() {
+
```

Comment:
Why is there empty code block here?

You shouldn't have an empty code block. If it needs to be empty for some reason, write a comment to justify why.

Same for those below.